Templates/CRMs/Magento/Magento Products to Salesforce/long-description.md

```diff
@@ -0,0 +1,21 @@
+This template inserts Magento products and their corresponding prices to Salesforce.
+Using this template, you can insert your new Magento products to Salesforce.
+
+![Template](assets/Adobe_Commerce___Magento_Product_to_Salesforce.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Salesforce user should have access to client ID, secret and security token.
+- Price book for Magento is initialized in Salesforce.
+
+# Implementation and Usage Notes
+
+This template uses Salesforce queries to identify a product and the price book it's connected to. Price book, in which the product price is inserted, can be changed by altering the price book task parameters.
```

Comment:
This sentence could probably be something like "The name of the price book in which the product prices are inserted, can be changed .... "

Reply:
Technically we change the whole price book, not just the name, but this gave me an idea 😄 "The price book, in which the product price is inserted, can be changed by altering the name in the price book task parameters."